modules/gapi/include/opencv2/gapi/imgproc.hpp

```diff
@@ -672,7 +672,6 @@ with the following \f$3 \times 3\f$ aperture:
 @note Function textual ID is "org.opencv.imgproc.filters.laplacian"
 
 @param src Source image.
-@param dst Destination image of the same size and the same number of channels as src .
```

Comment:
Oh, and actually if you do want to specify what your kernel returns - you can use the @return which looks like this:
<img width="398" alt="prscr_return" src="https://user-images.githubusercontent.com/43438132/79495537-697e7400-802d-11ea-862b-08d142e35e86.png">

Reply:
Fixed.